src/app/Types.elm

```diff
@@ -51,3 +53,4 @@ type Msg
     | SetCup CupOfDice
     | NextServerState ServerState
     | RoundFinished RoundResult
+    | TableMsg Table.Msg
```

Comment:
`Types.Msg` isn't meant to be the top, I think `Types.SocketMsg` is more appropriate then `Msg` can have SocketMsg and TableMsg.